x/ibc/spec/README.md

```diff
@@ -20,9 +20,9 @@ For the general specification please refer to the [Interchain Standards](https:/
 1. **[Concepts](01_concepts.md)**
 2. **[State](02_state.md)**
 3. **[State Transitions](02_state_transitions.md)**
```

Comment:
```suggestion
3. **[State Transitions](03_state_transitions.md)**
```

Reply:
this file doesn't exist. Should I create it?

Reply:
I added the file